module/localisation/RobotParticleLocalisation/data/config/RobotParticleLocalisation.yaml

```diff
@@ -1,8 +1,9 @@
 process_noise_diagonal: [0.001, 0.001, 0.0001]
-start_state: [-2, -3, 1.5]
-start_variance: [0.001, 0.001, 0.0000001]
 
-draw_particles: 20
+start_variance: [0.001, 0.001, 0.0000001] # Covariance
```

Comment:
that's reaaaaal low
```suggestion
start_variance: [0.001, 0.001, 0.0000001] # Covariance
```

Reply:
Seems fine
> Real value: 0.0000001
> float.....: 1e-07
> double....: 1e-07

Reply:
I guess I am wary because around 1e-8+ covariance was where the cholesky decomps started failing way more